README.md

```diff
@@ -153,19 +153,6 @@ where you specify the IP of the camera you want to proxy as the `MSP_CAMERA`
 environment variable (default is `192.168.0.90`). The webpack dev server will
 proxy requests to the camera, so that you'll have no CORS issues for any format.
 
-### Streaming without RTSP
-
-The function `fetchTransformationMatrix` can be used to place things
```

Comment:
Was the function removed as well? If not, maybe it should if the docs/purpose are gone too.

Reply:
I removed it

Reply:
Alright, in this case we also need to adjust the commit message to reflect a breaking change, since we no longer offer that function.

Reply:
Fixed